git.lua

```diff
@@ -4,8 +4,14 @@ local path = require('path')
 local git = require('gitutil')
 local matchers = require('matchers')
 local w = require('tables').wrap
+local ver = require('version')
+local color = require('color')
 local parser = clink.arg.new_parser
 
+if ver.supports_color_settings() then
```

Comment:
Could you rename the variable (and perhaps module as well) to `clink_version` - IMHO `clink_version.supports_color_settings()` reads slightly more natural

Reply:
Yes.